apps/src/lib/script-editor/LevelNameInput.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import FontAwesome from '../../templates/FontAwesome';
```

Comment:
super nit: i always prefer absolute paths for imports since it makes moving files around easier

Reply:
I think we are kind of split as a team on this -- I try to only use absolute paths when importing source files from tests or from entry points, where the relative path would get really long.

For whatever reason, my IDE can't resolve absolute paths when jumping to definition, but it can do it when relative paths are used. Further, it's pretty good about fixing relative paths when I refactor code from one file to another.

I see your point though that without the help of an IDE the relative paths are easier.